Cargo.toml

```diff
@@ -14,28 +14,22 @@ path = "src/main.rs"
 
 [dependencies]
 anyhow = "1.0"
-serde = "1.0.112"
-serde_derive = "1.0.112"
-serde_json = "1.0"
-serde_yaml = "0.8"
-libc = "0.2"
+bincode = "1.3.1"
+chrono = { version = "0.4.11", features = ["serde"] }
+clap = "~2.33"
 fs2 = "0.4.3"
+hex = "0.4.2"
+libc = "^0.2"
+libsystemd = "^0.2"
 nix = "0.17.0"
-tempfile = "3.0.3"
-clap = "~2.33"
-structopt = "0.3"
 openat = "0.1.19"
 openat-ext = "0.1.0"
 openssl = "^0.10"
-libsystemd = "^0.2"
-hex = "0.4.2"
-bincode = "1.3.1"
-bs58 = "0.3.1"
-byteorder = "1"
-c_utf8 = "0.1.0"
-lazy_static = "1.1.0"
-serde_plain = "0.3.0"
-chrono = { version = "0.4.11", features = ["serde"] }
+serde = { version = "^1.0", features = ["derive"] }
+serde_derive = "^1.0"
```

Comment:
This is probably redundant given that the `serde` entry above enables the `derive` feature already, I guess?

Reply:
Trying to drop that gives me failures, I simplified it the other way.

Reply:
I was curious/unsure about what you were seeing here, so I tried to quickly reproduce it but I didn't manage to. https://github.com/coreos/bootupd/pull/10 is an happy by-product of that.